cob_hardware_config/cob4-5/config/local_costmap_params.yaml

```diff
@@ -13,19 +12,54 @@ height: 5.0
 resolution: 0.07
 
 # footprint and range
-obstacle_range: 3.0 #[m]
-max_obstacle_height: 2.0 #[m]
-raytrace_range: 3.0 #3.0 #[m]
-#footprint: [[0.4,0.0],[0.35,0.35],[-0.2,0.4],[-0.45,0.0],[-0.2,-0.4],[0.35,-0.35]] #[m]
 footprint: [[0.43, 0.35], [0.43, -0.35], [-0.43, -0.35],[-0.43, 0.35]] #[m]
 footprint_padding: 0.02 #[m]
-inflation_radius: 0.55 #[m]
 
-# sensor setup
-observation_sources: laser_scan_front laser_scan_left laser_scan_right
-laser_scan_front: {sensor_frame: /base_laser_front_link, topic: base_laser_front/scan, data_type: LaserScan, marking: true, clearing: true}
+#layers
+plugins:
 
-laser_scan_left: {sensor_frame: /base_laser_left_link, topic: base_laser_left/scan, data_type: LaserScan, marking: true, clearing: true}
+# The obstacle layer tracks the obstacles as read by the sensor data.  The ObstacleCostmapPlugin marks and
+# raytraces obstacles in two dimensions, while the VoxelCostmapPlugin does so in three dimensions.
+- {name: obstacle_layer,  type: "costmap_2d::VoxelLayer"}
```

Comment:
A `costmap_2d::ObstacleLayer` should be enough here...
Also, please remember issue ipa320/ipa_navigation#67 which points out some issues to be improved (will be done after ipa320/ipa_navigation#66 has been merged).